examples/vacationsApprovalServerless/README.md

```diff
@@ -0,0 +1,86 @@
+# Vacation Approval App example using Serverless Framework
+
+With Smartmate you have the abillity to call custom endpoints in order to have manage custom logic outside the Smartmate enviroment.
+
+This example explains how to setup a Smartmate App using Serverless Framework.
+
+
+`Note`: With Serveless Framework you can use many Cloud Providers, in this case we will be using AWS.
+## Setup Serveless
+
+In order to setup a Serverless funtion, first you need to install the Serverless Framework:
+```
+   npm install -g serverless
+```
+
+Next install dependencies:
+```
+npm install
+```
+
+Then you need to create an `IAM USER and Access Key` in your AWS Console, for this you can follow the steps describe in this link: https://www.serverless.com/framework/docs/providers/aws/guide/credentials/
+
+The next step is to setup your credentials in the serveless cli:
+```
+serverless config credentials --provider aws --key <YOUR ACCESS KEY> --secret <YOUR SECREET ACCESS KEY>
+```
+Now you can edit the file `serverless.yml` if you want to add more functions.
+We will upload our function called `calculatedDaysOff`. This funciton recives two dates like query strings and calculated the difference in days beetween the two dates.
+
+You can see the code in the `handler.js` file.
+
+#### Test the Funtion locally
+
+To run the function locally run the following command in the app root
+```
+serverless offline 
+```
+or
+```
+sls offline
+```
+The function will be listening by default in this direction `http://localhost:3000/dev/daysoff`
+
+Now you can test it with Postman or from a browser.
+`Note`: You can change the path in the `serverless.yml` file.
+
+#### Deploy the Serverless Function
+In order to deploy the funtion to AWS you can run:
+```
+sls deploy
+```
+When the process will be done, in the cli you can see the deployed endpoint.
+You need to copy this endpoint for call it from a Service Task in the Smartmate Process.
+
+If you need to look this information again, you can run:
+```
+sls info
+```
+
+## Setup Smartmate Process
+
+Now you can go to the `processes/requestVacations.yml` file.
```

Comment:
Add something like:

Before starting, please take a look at the [Getting Started](https://docs.apps.smartmate.io/getting-started/index.html) page. Once you have your workspace set up, you can add this app by running: `sm add app` from insider your workspace, and selecting this app.